batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDao.java

```diff
@@ -0,0 +1,137 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.util.DBConnection;
+
+import javax.xml.crypto.Data;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Romit Amgai <romitamgai@lftechnology.com> on 1/19/16.
+ */
+public class EmployeeDao {
+  private Connection con;
+
+  public void insert(Employee employee) throws DataException {
+    try {
+      con = DBConnection.getSqlConnection();
+      String qry = "INSERT INTO employee (username,address,email,contact) VALUES (?,?,?,?)";
+      PreparedStatement preStmt = con.prepareStatement(qry);
+      preStmt.setString(1, employee.getName());
+      preStmt.setString(2, employee.getAddress());
+      preStmt.setString(3, employee.getEmail());
+      preStmt.setString(4, employee.getContact());
+      preStmt.execute();
+      preStmt.close();
+      con.close();
+    } catch (SQLException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)